src/test/java/org/fundacionjala/coding/dennis/EanCodeTest.java

```diff
@@ -0,0 +1,84 @@
+package org.fundacionjala.coding.dennis;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ *
+ */
+public class EanCodeTest {
+
+    /**
+     *
+     */
+    @Test
+    public void testEnCodeTrue() {
+        String code = "4003301018398";
+        assertTrue("It has to give True", EanCode.validate(code));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "It has to give True" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)